src/docs/Components/Buttons/code.mdx

```diff
@@ -122,33 +122,42 @@ To manage display of your button, you have 2 availables classes :
 
 You can add an icon into any buttons you want.
 
-You should add 2 CSS Classes to your icon HTML tag :
+You can add 2 CSS Classes to your icon HTML tag :
 
 - Element class : `mc-button__icon`
 - Modifier element class : `mc-button__icon--[size]`
```

Comment:
?

Reply:
it's not what I'm reading in the usage docs...

Reply:
fixed